Cargo.lock

```diff
@@ -2512,19 +2512,20 @@ dependencies = [
 
 [[package]]
 name = "minidump"
-version = "0.22.2"
+version = "0.26.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ebc7268c0afe5aa2fd6fde310e6cda158f0b3581f317c5a5c7d170861f9b90a6"
+checksum = "ee9ea21482e519a57bfc5df90b736f25465ef349a31c18ff2c6332a2f18474de"
 dependencies = [
  "debugid",
  "encoding_rs",
  "memmap2",
- "minidump-common",
+ "minidump-common 0.26.0",
```

Comment:
We now have 2 copies of this crate in the dependency tree, please update everything that needs it. Most likely you missed something in `relay-prosperoconv`.

Reply:
`relay-prosperoconv` has a `minidump-writer` dependency but the latest version of that uses `"minidump-common 0.24.1"`

Reply:
As discussed, this is a separate crate and has a different lifecycle, so we can't 'fix' this.